mhs/selenium_tests/page_objects/logs_retriever.py

```diff
@@ -0,0 +1,84 @@
+import enum
+import os
+from operator import itemgetter
+from typing import List, Generator, AnyStr
+
+import boto3
+
+from definitions import ROOT_DIR
+
+
+class Component(enum.Enum):
+    MHS = enum.auto()
+    SCR = enum.auto()
+
+
+_component_log_group_name_map = {
+    Component.MHS: "/ecs/test-environment",
+    Component.SCR: "/ecs/scr-service-environment",
+}
+
+_component_log_filename_map = {
+    Component.MHS: "mhs.txt",
+    Component.SCR: "scr.txt"
+}
+
+
+def get_logs(component: Component) -> Generator:
+    if os.environ.get("MHS_ADDRESS", "localhost") == "localhost":
+        return _get_logs_from_file(component)
+    return _get_logs_from_aws_cloud_watch(component)
+
+
+def get_logs_as_list(component: Component) -> List[AnyStr]:
+    return list(get_logs(component))
+
+
+def _get_logs_from_file(component: Component) -> Generator:
```

Comment:
Optional: could be more specific as to what types the generator returns, or maybe just do `Iterator[str]` (I'm basing this on what [the docs say](https://docs.python.org/3.7/library/typing.html#typing.Generator))